turbo.json

```diff
@@ -81,11 +81,11 @@
         },
         "test:unit:browser": {
             "dependsOn": ["^compile:js"],
-            "inputs": ["$TURBO_DEFAULT$", "src/**"]
+            "inputs": ["$TURBO_DEFAULT$", "../../.agave/**/version.yml", "src/**"]
```

Comment:
K don't even get me started on tooling _not_ following symlinks (I couldn't write `active_release/` here; had to write `**/`).

Reply:
But wait, does this work? Won't two asterisks match `version.yml` across multiple versions? Why doesn't Turbo recognize the symlink?

Reply:
It will only work if we can reliably assume our CI never installs more than one version. Which is true now, but will surely break one day.

Reply:
You're right! In any case, it's likely that a change in the set of `versions.yml` will imply that you installed a _newer_ version and are using that now.